app/templates/apps/new.html

```diff
@@ -4,39 +4,54 @@
 
 {% block main_column %}
 
-<form action="{{ url_for('apps.create') }}" method="post">
+<div class="text">
+  <p class="lede form-group">After <a href="https://moj-analytical-services.github.io/platform_user_guidance/deploying-a-shiny-app.html">creating an app in Github</a>, use this form to register your app and connect it to sensitive data stored in S3.</p>
+</div>
+
 
+<form action="{{ url_for('apps.create') }}" method="post">
   <div class="form-group {% if errors.name %} form-group-error {% endif %}">
-    <label for="name" class="form-label">Name</label>
+    <label for="name" class="heading-medium form-label">
+      Github repo url
```

Comment:
Minor wording question, shouldn't be 'URL' instead of 'url'?

Reply:
👍

Reply:
/me waiting for punch from the side :P